### PR TITLE
Update multi-device document

### PR DESCRIPTION
**Pull request by xunzhang**

### CODE REVIEW

File: doc/source/commandline.rst [hunk 1]
```diff
@@ -28,6 +28,8 @@ options.
    `here <http://dynet.readthedocs.io/en/latest/minibatch.html>`_.
 -  ``--dynet-gpus NUMBER``: Specify how many GPUs you want to use, if
    DyNet is compiled with CUDA. Currently, only one GPU is supported.
+-  ``--dynet-gpu``: Specify whether to use GPU or not. Note that it is the only option
```
Comment (neubig) [inline doc/source/commandline.rst]: Do you mean "this is only an option for Python programs"? Is there a reason why we can't support it for C++ also?

Comment (xunzhang) [reply doc/source/commandline.rst]: I've no idea actually. The old version of DyNet only has this option in the Python layer. I just follow the previous rule(https://github.com/clab/dynet/blob/9606e2a3fdc31788b3739adfcfb498b1626645fd/python/dynet.py#L5).

**Comment by xunzhang** [comment]
It seems this blocks the readthedocs build: https://readthedocs.org/projects/dynet/builds/6012492/ when building `python_ref`. Let me fix it.

**Comment by pmichel31415** [comment]
I can take a look at readthedocs @xunzhang

**Comment by pmichel31415** [comment]
I think it's just that you are using `autoclass` for a function (`to_device`).

I will fix directly on the master branch

**Comment by xunzhang** [comment]
@pmichel31415 Oh right. Thanks a lot!

**Pull request merged**